source/process/guest-article.rst

```diff
@@ -0,0 +1,21 @@
+Guest Articles for Mattermost Apps and Services
+===============================================================
+
+This document outlines some of the procedures in developing guest articles. Mattermost, Inc. is excited to bring attention to guest company creators and their experiences with using Mattermost.
+
+**Article Writer:**
```

Comment:
Can we make it clear that this is someone inside Mattermost?

In case studies doc, we call it `Mattermost case study coordinator`, maybe we can say `Mattermost guest post coordinator` or something.